FitGoal/FitGoal/Setting/UserInfoCell.swift

```diff
@@ -87,4 +123,41 @@ class UserInfoCell: UITableViewCell {
             userInformation.centerYAnchor.constraint(lessThanOrEqualTo: self.contentView.centerYAnchor)
         ])
     }
+    
+    private func setPickerViewConstraints() {
+        genderPickerView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            genderPickerView.heightAnchor.constraint(equalToConstant: 150)
+        ])
+    }
+}
+
+extension UserInfoCell {
```

Comment:
When you implement a protocol in an extension, it helps indicating what protocols you are implementing. For example, in this case it would be better to declare the extension like:

```swift
extension UserInfoCell:  UIPickerViewDelegate, UIPickerViewDataSource {
    ...
}
```

And then remove the protocols conformance from the `UserInfoCell` type definition. Specifically:

```diff
- class UserInfoCell: UITableViewCell, UIPickerViewDelegate, UIPickerViewDataSource
+ class UserInfoCell: UITableViewCell
```